src/test/kotlin/racing/CarRacingKoTest.kt

```diff
@@ -1,38 +1,49 @@
 package racing
 
+import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.core.spec.style.DescribeSpec
 import io.kotest.matchers.shouldBe
-import java.io.ByteArrayOutputStream
-import java.io.PrintStream
+import racing.domain.Car
+import racing.domain.CarRacing
+import racing.domain.RacingCars
+import racing.domain.StaticPowerStrategy
+
+private fun testCar(name: String, position: Int = 0) =
+    Car(name = name, powerStrategy = StaticPowerStrategy(power = 10), startPosition = position)
 
 class CarRacingKoTest : DescribeSpec({
     describe("run method") {
-        val output = ByteArrayOutputStream()
-        System.setOut(PrintStream(output))
-
-        val expect =
-            """
-                |Andy : -
-                |Bruce : -
-                |
-                |Andy : --
-                |Bruce : --
-                |
-                |Andy : ---
-                |Bruce : ---
-                |
-                |Andy, Bruce가 최종 우승했습니다.
-                |
-            """.trimMargin()
-
-        context("both cars ( Andy, Bruce ) move by moveCount") {
-            it("draw car name and location for each turn, and draw the winner at the end") {
+
+        context("with move count less than 0 or not number") {
+            it("throws IllegalArgumentException") {
+                shouldThrow<IllegalArgumentException> {
+                    CarRacing(
+                        cars = RacingCars(listOf(testCar("Andy"), testCar("Bruce"))),
+                        moveCount = -1
+                    ).run()
```

Comment:
```suggestion
                    )
```
run 은 호출할 필요 없어 보이네요. 이렇게 되면 DCI 도 재정의가 필요할듯?